modules/abstract-utxo/src/parseOutput.ts

```diff
@@ -228,14 +228,20 @@ export async function parseOutput({
      * details are fetched on the wallet, a local address validation is run, whose errors however are generated
      * client-side and can therefore be analyzed with more granularity and type checking.
      */
-    const addressDetails = await fetchAddressDetails({
-      reqId,
-      addressDetailsVerification,
-      addressDetailsPrebuild,
-      currentAddress,
-      disableNetworking,
-      wallet,
-    });
+    const isCurrentAddressInRecipients = txParams.recipients?.some((recipient) =>
+      recipient.address.includes(currentAddress)
+    );
```

Comment:
The rest of this func doesn't do anything useful if this is `false`. I believe we should just `return currentOutput` in that case.

Reply:
Tried this but run into test failure where we don't decorate a change address with external true if I return early.

Reply:
Skipped the test for now since test expects to decorate an output when the address is part of the recipients which is no longer the case with this chance. Added a TODO to unskip.